src/UserSpaceInstrumentation/MachineCode.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef USER_SPACE_INSTRUMENTATION_MACHINE_CODE_TOOLING_H
```

Comment:
Change the include guard to `USER_SPACE_INSTRUMENTATION_MACHINE_CODE_H_`.

Reply:
done